src/blueprints/ember-addon/__addonLocation__/babel.config.json

```diff
@@ -3,7 +3,7 @@
 <% } %>  "plugins": [
     "@embroider/addon-dev/template-colocation-plugin",<% if (options.packages.addon.hasTypeScript) { %>
     ["@babel/plugin-transform-typescript", { "allowDeclareFields": true }],<% } %>
-    ["@babel/plugin-proposal-decorators", { "legacy": true }],
+    ["@babel/plugin-proposal-decorators", { "version": "legacy" }],
```

Comment:
I tested `{ "version": "legacy" }` with `ember-container-query` as well as 4 private addons. The tests for these addons passed.